lib/data/network/contact/tom_contact_api.dart

```diff
@@ -35,4 +36,11 @@ class TomContactAPI {
 
     return LookupMxidResponse.fromJson(response);
   }
+
+  Future<HashDetailsResponse> getHashDetails() async {
+    final response = await _client.postToGetBody(
+      IdentityEndpoint.hashDetailsServicePath.generateMatrixIdentityEndpoint(),
+    );
+    return HashDetailsResponse.fromJson(response);
+  }
```

Comment:
srr, but it should belong to identity server?

Reply:
done